### ISSUE TITLE
Can the SHiPS assembly be packaged for NuGet?

### ISSUE THREAD

**Issue by Jaykul** [PowerShell/SHiPS#101]
It would be really useful for binary authors if we could just add a nuget reference to get the assembly.

Even if we still needed the SHiPS module, the [current instructions](https://github.com/PowerShell/SHiPS/tree/development/samples/FamilyTreeInCSharp) require us to compile SHiPS -- which seems really unnecessary.

**Comment by jianyunt** [comment]
Good idea to publish the SHiPS assemblies to nuget. Yes compiling SHiPS is the way and we can always install-module SHiPS to get the assembly.

**Comment by DarqueWarrior** [comment]
I am facing this right now. I have been using SHiPS with VSTeam (pure Script module) and starting porting more of VSTeam to a binary module. Creating a Provider from the ground up will be a pain. I would just like to port the classes from ps1 to cs files using the same Ships directory and leaf base classes.
